### DISCUSSION TITLE
v1.4.0

### DISCUSSION THREAD

**Discussion by gustavo-iniguez-goya** [evilsocket/opensnitch#495] [Releases]
What's new
---

* Added eBPF to intercept processes.
* Added nftables support.
* Added block/allow lists to block ads, spam, etc [read more](https://github.com/evilsocket/opensnitch/wiki/block-lists).
* Ability to filter connections from containers [see how](https://github.com/evilsocket/opensnitch/wiki/System-rules).
* GUI improvements:
   - More customizable.
   - Better performance.
   - Better integration with mobile devices.
   - Better handling of remote nodes.
   - Translated to 6 languages.

You can read more about all the changes following these links:
https://github.com/evilsocket/opensnitch/releases/tag/v1.4.0-rc.1
https://github.com/evilsocket/opensnitch/releases/tag/v1.4.0-rc.2
https://github.com/evilsocket/opensnitch/releases/tag/v1.4.0-rc.3
https://github.com/evilsocket/opensnitch/releases/tag/v1.4.0-rc.4

Don't forget the read the [FAQs](https://github.com/evilsocket/opensnitch/wiki/FAQs) and [known problems](https://github.com/evilsocket/opensnitch/wiki/Known-problems) before reporting an issue.

Downloads
---
#### daemon
- [deb x86_64](https://github.com/evilsocket/opensnitch/releases/download/v1.4.0/opensnitch_1.4.0-1_amd64.deb)
- [deb i386](https://github.com/evilsocket/opensnitch/releases/download/v1.4.0/opensnitch_1.4.0-1_i386.deb)
- [deb armhf](https://github.com/evilsocket/opensnitch/releases/download/v1.4.0/opensnitch_1.4.0-1_armhf.deb) (for raspberry pi and others)
- [deb arm64](https://github.com/evilsocket/opensnitch/releases/download/v1.4.0/opensnitch_1.4.0-1_arm64.deb) (for raspberry pi, Librem5, and others)
- [rpm x86_64](https://github.com/evilsocket/opensnitch/releases/download/v1.4.0/opensnitch-1.4.0-1.x86_64.rpm)

#### GUI
- [rpm](https://github.com/evilsocket/opensnitch/releases/download/v1.4.0/opensnitch-ui-1.4.0-1.f29.noarch.rpm)
- [deb](https://github.com/evilsocket/opensnitch/releases/download/v1.4.0/python3-opensnitch-ui_1.4.0-1_all.deb) 

(If the above packages complain about dependencies, use these ones)
- [deb - for old distributions](https://github.com/evilsocket/opensnitch/releases/download/v1.4.0/python3-opensnitch-ui_1.4.0-1_all.PIP.deb) (Ubuntu <= 18.04)
- [rpm - for old distributions](https://github.com/evilsocket/opensnitch/releases/download/v1.4.0/opensnitch-ui-1.4.0-1.noarch.rpm) (Fedora < 29)

<hr /><em>This discussion was created from the release <a href='https://github.com/evilsocket/opensnitch/releases/tag/v1.4.0'>v1.4.0</a>.</em>

**Comment by gustavo-iniguez-goya** [comment]
Thank you all who have made this version possible: @themighty1  @ovari  @shollingsworth  @NRGLine4Sec @freddii @Herve5 @Techtonictools  @tioguda @raboof @gnuhead-chieb etc, and the rest of you who have taken the time to submit and issue (and if you posted detailed information + logs ${GOD} bless you. If you completed the Bugs template I'll invite you a beer if you happen to come ${over here})

**Comment by Neo23x0** [comment]
I can't get the RPM for the GUI. The link seems to be broken.

**Reply by gustavo-iniguez-goya** [reply]
fixed, thank you for reporting it!
